src/components/PrimaryMap.tsx

```diff
@@ -430,6 +473,16 @@ export class PrimaryMap extends React.Component<Props, State> {
 
     this.map.on('pointermove', this.handleMouseMove)
     this.map.on('moveend', this.emitViewChange)
+
+    this.map.addEventListener('measureToolOpened', () => {
+      this.isMeasureToolInUse = true
```

Comment:
Can we attach this to the component's `this.state` instead of directly on `this`?  That way we can make sure it doesn't get lost during component updates.